stylesheets/scss/_ting.scss

```diff
@@ -5,24 +5,40 @@
   }
 }
 
+.pane-ting-collection {
+  .ting-collection-inner-wrapper {
+    .view-mode-collection-list {
+      &:last-child {
+        padding-bottom: 20px;
+      }
+    }
+  }
+}
+
 .view-mode-collection-list {
   @include content-box;
   .field-type-ting-type {
     display: none;
   }
   .field-name-ting-subjects {
+    float: left;
```

Comment:
As I see it adding floats will not fix alignment 100%. Current top margin of buttons cause problems.